src/main/java/io/burt/athena/AthenaDriverInfo.java

```diff
@@ -0,0 +1,42 @@
+package io.burt.athena;
+
+import java.io.IOException;
+import java.util.Properties;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class AthenaDriverInfo {
```

Comment:
This could be a package private class, there isn't any need to expose it unless we have to. Someone might start using it.

Reply:
Fixed.